pages/linux/i3status.md

```diff
@@ -0,0 +1,17 @@
+# i3status
+
+> Status line for the i3 window manager.
+> This command is usually called from the i3 configuration file.
+> More information: <https://i3wm.org/i3status/manpage.html>.
+
+- Periodically print the status line to stdout, using the default configuration:
+
+`i3status`
+
+- Periodically print the status line to stdout, using a specific consiguration:
```

Comment:
A typo
```suggestion
- Periodically print the status line to stdout, using a specific configuration:
```
What about starts with the `print` verb?
```suggestion
- Print the status line to stdout periodically, using a specific configuration:
```

Reply:
I'm not sure, it sounds a bit unnatural. Do you think we should change it?

Reply:
We try to start with a verb :man_shrugging: this example description is like the `watch` page.